edflow/iterators/template_iterator.py

```diff
@@ -46,8 +46,21 @@ def __init__(self, *args, **kwargs):
         self._log_ops = set_default(
             self.config, "log_ops", ["train/log_op", "validation/log_op"]
         )
+        default_minimal_logging = {
+            "active": True,
```

Comment:
keys active and handlers are never used afaik. Should be either removed or made functional.

Reply:
Making it functional is going to take me a little bit I guess. When implementing the feature I have struggled to figure out how to test new functionality.

Reply:
So I just removed them